### PR TITLE
Added  Flight Simulation Programming Server aka Coders Craft

### PR DESCRIPTION
**Pull request by Airbus5717**
That Server is useful for Flight Sim programming

### CODE REVIEW

**Comment by Airbus5717** [comment]
Added Another server for game development tool

File: README.md [hunk 1]
```diff
@@ -1534,14 +1541,21 @@ Language: English
...
 
 [__LinuxGSM__](https://discord.com/invite/UM4VTpK) [<img height="16px" width="16px" alt="Official Badge" src="images/badges/official.webp">](badges.md#official-identification-badge) [<img height="16px" width="16px" alt="Homepage URL" src="images/badges/homepage.webp">](https://linuxgsm.com/) [<img height="16px" width="16px" alt="Git Repository" src="images/badges/git.webp">](https://github.com/GameServerManagers      ) \
 Notable Channels: `#linuxgsm`, `#linux`, `#game-server`, `#general`, `#battlestations` \
+Language: English <br/> <br/>
+
+
+<img align="left" height="94px" width="94px" alt="Server Icon" src="https://pluckyporcupinegame.boutique/wp-content/uploads/2020/03/raylib_256x256.png" />
+
+[__Raylib__](https://discord.com/invite/VkzNHUE) [<img height="16px" width="16px" alt="Reddit Badge" src="images/badges/reddit.webp">](https://www.reddit.com/r/raylib/) [<img height="16px" width="16px" alt="Homepage URL" src="images/badges/homepage.webp">](https://www.raylib.com/) [<img height="16px" width="16px" alt="Git Repository" src="images/badges/git.webp">](https://github.com/raysan5/raylib) \
```
Comment (raysan5) [inline README.md:1555]: Probably raylib should be moved to a separate PR...

Comment (Airbus5717) [reply README.md:1555]: Yeah @mhxion should accept the PL

**Review by mhxion — COMMENTED** [review]
Hello @Airbus5717 Thanks for the PR and sorry for late reply. I have been holding up your PR since I haven't had any dedicated section for simulation development related communities, until [now](https://github.com/mhxion/awesome-discord-communities#simulation), and it wasn't a top priority. Your community meets the requirements and can be added once I add a subcategory specifically for flight simulation. If you don't mind, you need to wait a little longer, before I finally merge your PR.

**Comment by mhxion** [comment]
@Airbus5717 Due to the merge conflicts I had to manually resolve and add your community with some modifications. Feel free to open a new PR to make any new change.
